proposals/0016-c-interface-compiler-library.md

```diff
@@ -354,11 +374,326 @@ struct IDxcContainerReflection : public IUnknown {
 };
 ```
 
+ID3D12ShaderReflection provides access to descriptions and other information
+about the many different parts of the shader.
+
+* [Shader Description](#shader-description)
+* [Buffers](#shader-buffers-description)
+* [Variables and type information](#shader-types=and-variables)
+* [Functions and parameters](#shader-functions-and-parameters)
+
+PIX uses this interface to obtain nice buffer formatting behavior for the UI.
```

Comment:
PIX isn't the only user of this interface, and the buffer formatting behavior is just one possible use of it.  Suggest making it clearer in the wording that this is just an example scenario.

Reply:
Fixed